src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -0,0 +1,445 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Linq;
+using System.Threading;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Symbols.Metadata.PE;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Symbols
+{
+    public class DefaultInterfaceImplementationTests : CSharpTestBase
+    {
+        [Fact]
+        public void MethodImplementation_01()
+        {
+            var source1 =
+@"
+public interface I1
+{
+    void M1() 
+    {
+        System.Console.WriteLine(""M1"");
+    }
+}
+
+class Test1 : I1
+{}
+";
+            var compilation1 = CreateCompilationWithMscorlib(source1, options:TestOptions.DebugDll, 
+                                                             parseOptions: TestOptions.Regular.WithLanguageVersion(LanguageVersion.Latest));
+            Assert.True(compilation1.Assembly.RuntimeSupportsDefaultInterfaceImplementation);
+
+            var m1 = compilation1.GetMember<MethodSymbol>("I1.M1");
+
+            Assert.False(m1.IsAbstract);
+            Assert.True(m1.IsVirtual);
+
+            var test1 = compilation1.GetTypeByMetadataName("Test1");
+
+            Assert.Same(m1, test1.FindImplementationForInterfaceMember(m1));
+
+            compilation1.VerifyDiagnostics();
+            Assert.True(m1.IsMetadataVirtual());
+
+            CompileAndVerify(compilation1, verify:false,  
```

Comment:
> It looks like PEVerify fails, so it is skipped for now. Should we file a follow-up issue?

Regular runtime doesn't support the feature. Once we have runtime we can test against, new tests will have to be written. I will make it clear in the feature markdown file.